src/components/Modals/Receive/ReceiveInfo.tsx

```diff
@@ -179,8 +184,13 @@ export const ReceiveInfo = ({ asset, accountId }: ReceivePropsType) => {
               mt={8}
               bg='white'
             >
-              <LightMode>
-                <Card.Body display='inline-block' textAlign='center' p={6}>
+              <Card.Body display='inline-block' textAlign='center' p={6}>
+                <AccountDropdown
+                  assetId={asset.assetId}
+                  onChange={setAccountId}
+                  buttonProps={{ height: 5, variant: 'solid' }}
```

Comment:
We probably need to tweak this so the selected color isn't white, as at the moment it disappears when clicked:

<img width="291" alt="Screen Shot 2022-09-08 at 4 21 39 pm" src="https://user-images.githubusercontent.com/97164662/189049194-bcd4104e-c680-4d96-886e-611f5eef588b.png">

<img width="336" alt="Screen Shot 2022-09-08 at 4 22 08 pm" src="https://user-images.githubusercontent.com/97164662/189049241-70e54ad8-e59e-407d-8601-664eb2576016.png">

Reply:
yep this is an interesting one. It was white on white I juggled stuff a bit to get the dropdown outside of `LightMode` where it was mostly invisible. @reallybeard ? 🙏